features/step_definitions/animal_steps.rb

```diff
@@ -1 +1,19 @@
-#Delete this comment, here is where you should write your step defs
+Given(/^an animal$/) do
+  @bob = Animal.new('Bob', 'Cat', 5)
+end
+
+Then(/^His name is "(.*?)"/) do |name|
+  @bob.name == "Bob"
```

Comment:
You should check for equality against the value being passed in from the Gherkin script. You're reading what's already written in the feature.